docs/guides/customize-your-scene.mdx

```diff
@@ -0,0 +1,758 @@
+  const scene = await viewer.scene();
+
+  // Clear the overridden material on the item with the `item-uuid` ID
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearMaterialOverrides(),
+  ]);
+}
+```
+
+
+</details>
+
+#### `transform`
+
+- Parameters:
+  - `matrix: Matrix4x4` - the transformation matrix to use
+- Specifies that the items matching the query should have their transformation matrix overridden to match the specified transformation matrix.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+import { Matrix4, Vector3 } from '@vertexvis/geometry';
+
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Override the transformation matrix for the item with the `item-uuid` ID to
+  // move the element along the x-axis
+  await scene.items((op) => [
+    op
+      .where((q) => q.withItemId('item-uuid'))
+      .transform(Matrix4.makeTranslation(Vector3.create(100, 0, 0))),
+  ]);
+}
+```
+
+
+</details>
+
+#### `clearTransforms`
+
+- Parameters:
+  - `cascade?: boolean` - whether this operation should cascade to child items
+- Specifies that the items matching the query should have their overridden transformation matrix removed. The `cascade` flag determines whether
+  children of the items matching the query should also have their overridden transformation matrix removed, and defaults to `true`.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Clear the overridden the transformation matrix for the item with the `item-uuid` ID
+  // and do not cascade to preserve transformations on children
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearTransforms(false),
+  ]);
+
+  // Clear the overridden the transformation matrix for the item with the `item-uuid` ID
+  // and cascade to clear overridden transformations on children
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearTransforms(true),
+  ]);
+}
+```
+
+
+</details>
+
+#### `setPhantom`
+
+- Parameters:
+  - `phantomState?: boolean` - the phantom state to apply to the items
+- Specifies that the items matching the query should have their phantom state overridden to match the specified `phantomState` flag. If the
+  `phantomState` flag is not provided, it will default to `true`.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Mark the item with the `item-uuid` ID as phantom
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).setPhantom(true),
+  ]);
+
+  // Unmark the item with the `item-uuid` ID as phantom
+  await scene.items((op) => [
```

Comment:
Just another idea here, should we have the `setPhantom(false)` flow use the `.all()` query?

Reply:
Yeah, I included the `.all().clearPhantom()` example in the later fully-fledged phantom state example, but wanted to show that it can be explicitly marked false by ID as well here